### PR TITLE
BrandListing implementation related MR

### PR DESCRIPTION
**Pull request by Mansoor-Ahmed1201**

### CODE REVIEW

File: UI/paisa-kanaku/src/pages/setup/brand/Brand.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-
-const baseURL = "https://localhost:7122/v1/setup/brand/";
+import { getBrandList } from "../../../services/BrandServices";
 
 const Brand = () => {
   const columns = [
     {
       title: "Brand Name",
       dataIndex: "name",
-      key: 'id',
```
Comment (Praga-Dev) [inline UI/paisa-kanaku/src/pages/setup/brand/Brand.js:15]: avoid using double quotes inside Js files.

Comment (Mansoor-Ahmed1201) [reply UI/paisa-kanaku/src/pages/setup/brand/Brand.js:15]: > avoid using double quotes inside Js files.

Changes done. Please check

**Review by Praga-Dev — APPROVED** [review]
LGTM @Mansoor-Ahmed1201

**Comment by Praga-Dev** [comment]
Hi @Mansoor-Ahmed1201,
MR is done, Please verify your changes in Dev branch.

**Comment by Mansoor-Ahmed1201** [comment]
> Hi @Mansoor-Ahmed1201, MR is done, Please verify your changes in Dev branch.

Hi @Praga-Dev ,
Working fine

**Pull request merged**